overlord/registrystate/registrymgr_test.go

```diff
@@ -339,4 +332,27 @@ func (s *hookHandlerSuite) TestSaveViewHookErrorHoldsTasks(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(hooksup.Hook, Equals, "save-view-setup")
 	c.Assert(hooksup.Snap, Equals, "first-snap")
+
+	// the save-view hook for the second snap is made to wait for the rollback
+	nextHook := halts[0]
+	c.Assert(nextHook.Kind(), Equals, "run-hook")
+	c.Assert(nextHook.Status(), Equals, state.DoStatus)
+	err = nextHook.Get("hook-setup", &hooksup)
+	c.Assert(err, IsNil)
+	c.Assert(hooksup.Hook, Equals, "save-view-setup")
+	c.Assert(hooksup.Snap, Equals, "second-snap")
+	c.Assert(nextHook.WaitTasks(), HasLen, 2)
+	c.Assert(nextHook.WaitTasks()[1].ID(), Equals, rollbackTask.ID())
+}
+
+func (s *registryTestSuite) TestManagerOk(c *C) {
+	runner := s.o.TaskRunner()
+	hookMgr, err := hookstate.Manager(s.state, runner)
+	c.Assert(err, IsNil)
+
+	mgr := registrystate.Manager(s.state, hookMgr, runner)
+	s.o.AddManager(mgr)
+
+	err = s.o.Settle(5 * time.Second)
+	c.Assert(err, IsNil)
```

Comment:
does this bit do something?

Reply:
It just runs the manager's Ensure which is a no-op